OAuth/ResourceOwner/Auth0ResourceOwner.php

```diff
@@ -34,19 +34,35 @@ class Auth0ResourceOwner extends GenericOAuth2ResourceOwner
         'profilepicture' => 'picture',
     );
 
+    protected function getRequestHeaders($extends = array()) {
+
+        return array_merge($extends, array(
+            'User-Agent' => 'PHP/' . phpversion(),
+            'Auth0-Client' => 'PHP/HWIOAuthBundle',
```

Comment:
oh, didn't see that. Anyway, it only adds the User-Agent header with HWIOAuthBundle but we are adding some different info in a different format.

First, we are using the User-Agent to track the platform and version (in this case PHP/5.5.X for example) and in the Auth0-Client header some info about the SDK used.

It would be great if we can change this behaviour that allow us track better how our users consume the webservices (and also allow us to improve their experience by tracking deprecated sdks etc).

2 things about this PR:
- The way we send the headers will change a little so it shouldnt be merged yet
- It would be nice if we can add to the headers the package version (for example a constant like HWIOAuthBundle::VERSION). 

Do you think that is posible to make this changes?